spec/unacked_queue_spec.cr

```diff
@@ -0,0 +1,24 @@
+require "./spec_helper"
+
```

Comment:
I think we should have a "keeps track of bytesize" test here as well, similar to what we have in ReadyQueue, to ensure we always test subtraction during delete etc.